src/cloudant/database.py

```diff
@@ -299,3 +305,151 @@ def __iter__(self, remote=True):
                     yield doc
 
             raise StopIteration
+
+    def security_document(self):
```

Comment:
It feels a little weird to expose security_url as a public property, but also have this, since the user can write doc = mydb[self.security_url] in as little code as it would take to call this function.